README.md

```diff
@@ -18,7 +18,9 @@ build tooling will attempt to compile a static library from local Rust sources.
 To opt out of downloading precompiled assets, set `FFI_BUILD_FROM_SOURCE=1`:
 
 ```shell
-FFI_BUILD_FROM_SOURCE=1 make clean all
+rm .install-filecoin \
+    ; make clean \
```

Comment:
just `make clean` should be fine, right?

Reply:
I thought :) but it isn't. It will clean, but say "nothing to be done" on the next run it will then actually do the compilation. I couldn't be bothered digging deeper :)